src/server/cmd/pachtf/main.go

```diff
@@ -75,8 +90,14 @@ func sqlGenQueries(ctx context.Context, log *logrus.Logger, args []string) error
 		return errors.Errorf("must provide query")
 	}
 	query := args[0]
-	inputDir := filepath.FromSlash(pfs + "/in")
-	outputDir := filepath.FromSlash(pfsOut)
+	inputDir, err := filepath.EvalSymlinks(filepath.FromSlash(pfs + "/in"))
```

Comment:
what's the case where these paths would need symlinks resolved?

Reply:
The input subdirectories (the immediate children of `/pfs/`) are symlinks.  `filepath.WalkDir` doesn't follow symlinks.